### PR TITLE
Required vars

### PR DESCRIPTION
**Pull request by jdyke**
Added a note to the `org_id` and `folder_id` vars that one or the other is required for a simple deployment. Open to suggestions on better ways to notate the requirements.

### CODE REVIEW

**Comment by jdyke** [comment]
This PR was opened in response to #244

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Simple usage of the module within your own main.tf file is as follows:
 | external\_project\_access\_violations\_should\_notify | Notify for External Project Access violations | string | `"true"` | no |
 | firewall\_rule\_enabled | Firewall rule scanner enabled. | string | `"true"` | no |
 | firewall\_rule\_violations\_should\_notify | Notify for Firewall rule violations | string | `"true"` | no |
-| folder\_id | GCP Folder that the Forseti project will be deployed into | string | `""` | no |
+| folder\_id | GCP Folder that the Forseti project will be deployed into | string | `""` | no** |
```
Comment (morgante) [inline README.md]: This table is autogenerated, so we shouldn't make manual annotations.

I'd suggest we simply add a note at the TOP of the inputs table.

Comment (jdyke) [reply README.md]: You got it, will update

Comment (jdyke) [reply README.md]: I added a note. Let me know if we should change the wording.

Thanks

**Pull request merged**